charts/newrelic-logging/templates/daemonset-windows.yaml

```diff
@@ -43,6 +43,9 @@ spec:
       {{- if $.Values.image.pullSecrets }}
       imagePullSecrets:
 {{ toYaml $.Values.image.pullSecrets | indent 8 }}
+      {{- end }}
+      {{- with include "newrelic.common.priorityClassName" . }}
```

Comment:
Hi! This should be `{{- with include "newrelic.common.priorityClassName" $ }}` instead as for windows daemonset we're iterating over versions

Reply:
Thanks, I'll add the changes in the next days

Reply:
@danybmx Just pushed a new version, I also saw that there is already a priorityClassName set, but not from "commons" so I removed that one on windows.